app/assets/javascripts/maps.js

```diff
@@ -1,8 +1,11 @@
 var map;
 
-function initMap(x, y, image){
+function initMap(x, y, image, draggable, callback){
 	image = typeof image !== 'undefined' ? image : currentLocationImage;
 
+	// Draggable defaults to false
+	draggable = (draggable === undefined) ? false : true;
```

Comment:
Are we expecting `draggable` to be a boolean in general? Can this be:
```javascript
draggable = draggable || false;
```

Reply:
Thanks. Changed that. Also I think `maps.js` needs to be cleaned up a bit. There are lots of mixed tabs and spaces on the same line.